src/OrchardCore.Modules/OrchardCore.Seo/OrchardCore.Seo.csproj

```diff
@@ -30,6 +30,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Seo.Abstractions\OrchardCore.Seo.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Shortcodes.Abstractions\OrchardCore.Shortcodes.Abstractions.csproj" />
     <ProjectReference Include="..\OrchardCore.Indexing\OrchardCore.Indexing.csproj" />
+    <ProjectReference Include="..\OrchardCore.Media\OrchardCore.Media.csproj" />
```

Comment:
Let's not add this dependency. Two alternative ways to do this

First approach, move the `SeoMetaQueryObjectType` class into the Media module and register it using the following startup "in the Media startup

```
[RequireFeatures("OrchardCore.Apis.GraphQL", "OrchardCore.Seo")]
public class Startup : StartupBase
{
    public override void ConfigureServices(IServiceCollection services)
    {
        services.AddObjectGraphType<MetaEntry, MetaEntryQueryObjectType>();
        services.AddObjectGraphType<SeoMetaPart, SeoMetaQueryObjectType>();
    }
}
```

Second approach, Move both `MediaFieldQueryObjectType` and `MediaAssetObjectType` out of `OrchardCore.Media` and place them into `OrchardCore.Media.Core`. New the the SEO module can depend on  `OrchardCore.Media.Core` and you are good to go.

However, I am thinking to combine the 2 solutions. The first solution is better but at the same time we should also move `MediaFieldQueryObjectType` and `MediaAssetObjectType` out of `OrchardCore.Media` and place them into `OrchardCore.Media.Core`.

Reply:
ok, but want to know what is the reasoning, why shouldn't it depend on media when it's already dependant with manifest?

Reply:
In OC, we do not allow a module to depend on another module. It's okay for a module to depend on core projects but not a module.